opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/pollerd.adoc

```diff
@@ -0,0 +1,25 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[ga-opennms-operation-daemon-config-files-pollerd]]
+==== Pollerd
+
+[options="header, autowidth"]
+|===
+| Internal Daemon Name | Reload Event
+| _Pollerd_            | `uei.opennms.org/internal/reloadDaemonConfig -p 'daemonName Pollerd'`
+|===
+
+.Pollerd configuration file overview
+[options="header, autowidth"]
+|===
+| File                        | Restart Required | Reload Event | Description
+| `poller-configuration.xml`  | yes              | yes          | It is the main configuration file for _Pollerd_, and it can be reloaded using _reloadDaemonConfig_.
+                                                                  Restart is required in case new monitors are created or removed.
+                                                                  Reload Event loads changed configuration parameters of existing monitors.
+| `response-graph.properties` | no               | no           | It contains the graph templates for the metrics collected by _Pollerd_, but the file itself is not managed by _Pollerd_.
+                                                                  Any graph related file is reloaded automatically, every time it is changed.
+| `poll-outages.xml`          | ?                | ?            | It is related with the scheduled outages feature.
+| `poller-config.properties`  | ?                | ?            | ?
```

Comment:
`poll-outages.xml` can be reloaded using `uei.opennms.org/internal/schedOutagesChanged` as described in this thread http://opennms.530661.n2.nabble.com/Automated-Scheduled-outages-td6295452.html